### PR TITLE
New sidebar navigation

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description

This PR brings navigation UI updates to the left sidebar. Links are now consolidated into one section and some are hidden behind "more" link. It also brings new open source Twemoji for these links instead of what we used to use (license was a bit limiting).

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image

## Added tests?

- [x] no, because I need help

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

File: app/views/articles/_sidebar_nav.html.erb [hunk 1]
```diff
@@ -8,53 +8,105 @@
...
-          </div>
+          <% end %>
         <% end %>
       </div>
     </div>
-    <a href="<%= tags_path %>" class="cta cta-button">View all-time top tags</a>
-  </div>
+  </nav>
+
+  <% if user_signed_in? %>
```
Comment (benhalpern) [inline app/views/articles/_sidebar_nav.html.erb:87]: We should keep list of tags on home page in some capacity for SEO purposes. Help bots crawl to the tag pages more effectively.

**Comment by benhalpern** [comment]
One note about keeping tags on home page in some capacity for SEO, but otherwise looks good to me pending test updates.

File: app/views/articles/_sidebar_nav.html.erb (hunk 1 above)
Comment (ludwiczakpawel) [reply app/views/articles/_sidebar_nav.html.erb:87]: @benhalpern I’m not really touching tags section right now except moving “View all tags” link to the main nav

File: app/views/articles/_sidebar_nav.html.erb [hunk 2]
```diff
@@ -8,53 +8,105 @@
...
+      <a href="/about" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("rainbowdev.svg", aria: true, width: 24, height: 24, class: "crayons-icon crayons-icon--default p-1", title: "About") %>
+        About
+      </a>
+      <a href="/privacy" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/smart.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
+        Privacy policy
+      </a>
+      <a href="/terms" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/look.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
```
Comment (rhymes) [inline app/views/articles/_sidebar_nav.html.erb:68]: Title here should be "Terms of use"

File: app/views/articles/_sidebar_nav.html.erb [hunk 3]
```diff
@@ -8,53 +8,105 @@
...
+        Terms of use
+      </a>
+      <a href="/contact" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/contact.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
```
Comment (rhymes) [inline app/views/articles/_sidebar_nav.html.erb:72]: Title here should be "Contact"

File: app/views/articles/_sidebar_nav.html.erb [hunk 4]
```diff
@@ -8,53 +8,105 @@
...
+      <a href="/sponsors" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/heart.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Sponsors") %>
+        Sponsors
+      </a>
+      <a href="/about" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("rainbowdev.svg", aria: true, width: 24, height: 24, class: "crayons-icon crayons-icon--default p-1", title: "About") %>
+        About
+      </a>
+      <a href="/privacy" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/smart.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
```
Comment (rhymes) [inline app/views/articles/_sidebar_nav.html.erb:64]: Title here should be "Privacy policy"

**Comment by ludwiczakpawel** [comment]
@rhymes Good catches!

- I've fixed the titles
- I've shuffled order a little bit, now it's "Reading List, Listings, Podcasts, Videos, Tags" – I wanna keep Tags nav item visible by default because it's likely we will eventually get rid of (or move?) the section with tags below navigation. Even tough the `/tags` page isn't very useful now, I imagine it will be one day :)
- Hover text color should also be a little bit adjusted here and there. I do know there are elements with no background color on hover as we have in some other elements but that's a temporary thing until we adjust body background.

**Comment by ludwiczakpawel** [comment]
@rhymes I've removed the images but am having issues with tests... Could you pop in and look around when you have few mins?

**Comment by ludwiczakpawel** [comment]
@Ridhwana I brought your changes in. Could you take a quick look?

**Comment by Ridhwana** [comment]
> @Ridhwana I brought your changes in. Could you take a quick look?

@ludwiczakpawel it looks like my changes pulled in correctly. I have two questions though:
1. Is it correct that the color of the social media icons will be greyish?
2. The social media icons appear only appear when we click on "More" - I feel like the social media icons should always be there by default.

**Comment by ludwiczakpawel** [comment]
@Ridhwana 
1. This is correct. We're intentionally adjusting some colors.
2. This is fine too, I checked with Ben about that as well. We gonna add social icons in other places too later on so they have more exposure :). For now it's ok to kinda hide them a little.

**Comment by Ridhwana** [comment]
> @Ridhwana
> 
> 1. This is correct. We're intentionally adjusting some colors.
> 2. This is fine too, I checked with Ben about that as well. We gonna add social icons in other places too later on so they have more exposure :). For now it's ok to kinda hide them a little.

Awesomeness!! Thanks @ludwiczakpawel this looks great! Nice work 💥

**Pull request merged**